docs/_posts/examples/3400-01-23-adjust-layer-opacity.html

```diff
@@ -0,0 +1,86 @@
+---
+layout: example
+category: example
+title: "Adjust a layer's opacity"
+description: "Drag the range slider to adjust the opacity of the water layer in Mapbox streets."
```

Comment:
Can you update the description too? It's no longer the water layer being adjusted.